### ISSUE TITLE
error when clicking on product in front office to view details

### ISSUE THREAD

**Issue by kennethbruyninckx** [Oksydan/modern-prestashop-starter-theme#66]
PS 1.7.7.5 (which was the latest PS version around when I installed my webshop in July 2021, updating has always been a challange and a hit and miss for me... resulting in crashed and pretty unusable shops)
PHP 7.3.29
starter theme 1.3.0
is_imageslider-1.0.4
is_searchbar-1.0.0
is_shoppingcart-1.0.0
is_themecore-1.1.2

The theme was installed on a test copy of my live webshop installed in July 2021.
When I click on a products picture to see more details in front office I get the following:

![image](https://user-images.githubusercontent.com/63542736/147986995-88d42aaf-afba-41bf-8da6-5f44a1c7c587.png)

**Comment by kennethbruyninckx** [comment]
Another test copy of my shop was painstakingly (manually) upgraded to 1.7.8.2 and Starter theme v2.0.2 was installed on there with similar results as I click on a products picture to view more details:

![image](https://user-images.githubusercontent.com/63542736/147987429-94d5947d-7a1e-4340-9422-062cfb2c8d91.png)

**Comment by Oksydan** [comment]
Hi @kennethbruyninckx,
I am really sorry that it took me too long to response, I am really busy recently. 
Problem is related to php version 7.4 and it will be fixed with is_themecore version 2.0 (huge update complete module refacto.

You should be able to fix it be replacing this[ line ](https://github.com/Oksydan/is_themecore/blob/main/classes/ThemeStructuredJsonData.php#L268)
```php 
if ($product['show_condition'] && $product['condition']) {
```
with
```php
if (!empty($product['show_condition']) && !empty($product['condition'])) {
```

**Comment by caovillanueva** [comment]
i could fix this issue replacing the follow lines:

`` $conditions[$product['condition']]

with:

$product['condition']['type']

All the conditional should be something like :

if ($product['show_condition'] && $product['condition']) {
        $jsonData['offers'] = array_merge($jsonData['offers'], [
          'itemCondition' => $conditions[$product['condition']['type']]
        ]);
      }

**Comment by Oksydan** [comment]
Problems has been resolved on 2.0.0 is_themecore branch and it will be released soon with new version of theme.
